x-pack/plugin/security/qa/multi-cluster/src/javaRestTest/java/org/elasticsearch/xpack/remotecluster/RemoteClusterSecurityCrossClusterApiKeySigningIT.java

```diff
@@ -212,9 +220,19 @@ private void assertCrossClusterSearchSuccessfulWithoutResult() throws IOExceptio
     }
 
     private void assertCrossClusterSearchSuccessfulWithResult() throws IOException {
+        final long startTimeMillis = System.currentTimeMillis();
         boolean alsoSearchLocally = randomBoolean();
         final Response response = simpleCrossClusterSearch(alsoSearchLocally);
         assertOK(response);
+
+        try {
+            // Call the generic assertion function with the expected 'authentication_success' event
+            assertAuditLogContainsNewEvent(startTimeMillis, "authentication_success");
```

Comment:
There are [constants](https://github.com/elastic/elasticsearch/blob/main/x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/audit/AuditLevel.java#L14-L29) for `authentication_success` and `authentication_failed` in `AuditLevel` that would be nice to use here.